File: fixtures/sample.cs
using System;
using System.Collections.Generic;

namespace App.Services;

public sealed class Greeter<T> where T : IComparable<T>
{
    private const int MaxSize = 100;
    private readonly List<T> _items = new();

    public string Name { get; init; } = "greeter";

    public void Add(T item)
    {
        if (_items.Count >= MaxSize) throw new InvalidOperationException("full");
        _items.Add(item);
    }

    public async Task<int> FlushAsync()
    {
        await Task.Delay(10);
        return _items.Count;
    }
}

// Line comment
/* Block comment */
/// <summary>XML doc comment</summary>

File: fixtures/sample.py
import os
from typing import Optional

CONST = 42

class Greeter(Base):
    """Doc."""
    def __init__(self, name: str, count: Optional[int] = None) -> None:
        self.name = name
        self._count = count or CONST
    @property
    def label(self) -> str:
        return f"{self.name}: {self._count!r}"

def main(*args, **kwargs):
    g = Greeter("x")
    if g.label is not None and CONST > 3:
        print(g.label, os.sep)
    return [i**2 for i in range(10) if i % 2 == 0]

# Line comment: TODO rework this section
# another single-line comment
